### ISSUE TITLE
Support attribute wrappers & inline tags

### ISSUE THREAD

**Issue by marksiemers** [jeromegn/slang#26]
Any plans to support these two slim-lang features?
[Attributes wrappers](http://www.rubydoc.info/gems/slim/frames#Attributes_wrapper) (e.g. square brackets)
```slim
h2[id="tagline" class="small tagline"] = page_tagline
```
or [inline tags](http://www.rubydoc.info/gems/slim/frames#Inline_tags):
```slim
ul
  li.first: a href="/a" A link
  li: a href="/b" B link
```
or both:
```slim
ul
  li.first: a[href="/a"] A link
  li: a[href="/b"] B link
```

**Comment by jeromegn** [comment]
Yes. Well there's no roadmap really, but this would be nice. Initially, the plan was to implement those features, but I decided to dumb it down a bit to get something out there.

**Comment by jeromegn** [comment]
I've implemented attribute wrappers in 637e264d728f769db59f61b7707a3e828de45816 if you want to give that a shot.

I'll look into inline tags soon!

**Comment by marksiemers** [comment]
Thanks, amazingly quick turn-around.

I might add these (or some of them) to your fixtures (cause I'm paranoid without tests):
```slim
div [hello="world" foo="bar"]
div (hello="world" foo="bar")
div {hello="world" foo="bar"}
div {hello="#{"world"}" foo="bar"}

.hello[hello="world" foo="bar"]
.hello(hello="world" foo="bar")
.hello{hello="world" foo="bar"}
.hello{hello="#{"world"}" foo="bar"}

.hello [hello="world" foo="bar"]
.hello (hello="world" foo="bar")
.hello {hello="world" foo="bar"}
.hello {hello="#{"world"}" foo="bar"}

#hello[hello="world" foo="bar"]
#hello(hello="world" foo="bar")
#hello{hello="world" foo="bar"}
#hello{hello="#{"world"}" foo="bar"}

#hello [hello="world" foo="bar"]
#hello (hello="world" foo="bar")
#hello {hello="world" foo="bar"}
#hello {hello="#{"world"}" foo="bar"}
```

**Comment by jeromegn** [comment]
> Thanks, amazingly quick turn-around.

I can't say I've been consistent there, but I'm happy my schedule worked out for this one.

> I might add these (or some of them) to your fixtures (cause I'm paranoid without tests):

I don't know if we want to support whitespace like that. I can see adding this at some point, but right now it would add complexity and potentially some bugs.

I think it's acceptable to enforce no spaces there.

Slang is only _inspired_ by slim :)

> inline tags

As of 76a5954ab034939da25b18311cb94a96a794581e, Slang also support inline tags :)

Will cut a release soon! Going to fix a few more issues first.

**Comment by jeromegn** [comment]
Released as part of 1.7.0
